dev-games/ogre/ogre-2.1.ebuild

```diff
@@ -115,8 +134,16 @@ src_configure() {
 		-DOGRE_LEGACY_ANIMATIONS=$(usex legacy-animations)
 		-DOGRE_PROFILING_PROVIDER=$(usex profile none internal)
 		-DOGRE_USE_BOOST=no
+		-DOGRE_CONFIG_DOUBLE=no
 	)
 
+	# GLES2 is not supported, yet
+	#	-DOGRE_BUILD_COMPONENT_HLMS_PBS=$(         usex mobile no yes)
+	#	-DOGRE_BUILD_COMPONENT_HLMS_PBS_MOBILE=$(  usex mobile)
+	#	-DOGRE_BUILD_COMPONENT_HLMS_UNLIT=$(       usex mobile no yes)
+	#	-DOGRE_BUILD_COMPONENT_HLMS_UNLIT_MOBILE=$(usex mobile)
+	#	-DOGRE_BUILD_RENDERSYSTEM_GLES2=$(usex gles2)
+
 	# The double-precision mode can not be enabled, yet.
 	#	-DOGRE_CONFIG_DOUBLE=$(usex double-precision)
```

Comment:
Ogre wiki says you also need to
```
Disable OGRE_SIMD_NEON
Disable OGRE_SIMD_SSE2
```
Have you tried disabled them? If not, isn't this the cause for the failure to build with double precision enabled?

Reply:
I more hooked up on the very first sentence:
> We're not yet officially maintaining double precision, but it works (mostly?)

Well, double-precision already has a performance impact. Deactivating NEON/SSE2 for it to work doesn't sound wise to me.

However, I hit another problem. Running kernel-5.7+ means, that sysctl.h has been moved to /usr/inlcude/linux, and the build searches it in /usr/include/sys and fails. Dang.
This means I'll have to get a fix for kernels having it in there. AFAIR sys/sysctl.h is an Apple thing and irrelevant for Linux.

Give me a moment...

Reply:
Reported here: https://github.com/OGRECave/ogre-next/issues/132